x/session/keeper/session_hydrator.go

```diff
@@ -21,6 +21,7 @@ var SHA3HashLen = crypto.SHA3_256.Size()
 // TODO(#21): Make these configurable governance param
 const (
 	NumBlocksPerSession         = 4
+	SessionGracePeriod          = NumBlocksPerSession
```

Comment:
We will anyway end up converting it to blocks since we'll be using it's `blockHeight` representation when it's involved in `currentBlockHeight` comparisons or passing blockHeights to `waitForBlock` to create `Claim`s.

We'll be replacing `SessionGracePeriod` by `SessionGracePeriod * sessionkeeper.NumBlocksPerSession` in all its usage occurrences.

It also makes it non-trivial to switch to a block granular representation if we want to do so in the future IMO

Reply:
I want to push back on this.

The governance parameter (what we will document, update through governance votes, and what people read) will be in sessions. That is `SessionGracePeriod`.

Internally, we do math as a function of `SessionGracePeriod * NumBlocksPerSession`, so I think the separation will make maintenance simpler (especially if we leave the appropriate comments).

Lmk if jumping on a call is easier.